admin/client/App/sagas/index.js

```diff
@@ -8,8 +8,9 @@ import { loadItems } from '../screens/List/actions';
 /**
  * Load the items
```

Comment:
I don't understand why this function was renamed at all, it currently does two things:

1. Load the items
2. Update the query params

Both of which imo aren't expressed with `filterTheItems`. This should really be called `handleListChange` or something like that?